refactor(ast_codegen): flatten code

overlookmotel opened the pull request:
A bit of code style simplification after #4731. Remove unnecessary nesting and rename some vars for consistency.

overlookmotel commented:
* **#4743** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4743?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @overlookmotel and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

rzvxa commented:
I didn't add a merge label so #4737 goes through, I'm afraid that stack's madness might cause issues in the merge queue.

overlookmotel committed:
refactor(ast_codegen): flatten code (#4743)

A bit of code style simplification after #4731. Remove unnecessary nesting and rename some vars for consistency.
